integration/resource_lacework_vulnerability_exception_container_test.go

```diff
@@ -42,6 +42,9 @@ func TestVulnerabilityExceptionContainerCreate(t *testing.T) {
 	assert.Equal(t, "[CVE-2016-9840 CVE-2018-14599 CVE-2018-6942]", actualCves)
 	assert.Equal(t, "[map[name:adm-zip version:0.4.7] map[name:myPackage version:1.0.0]]", actualPackages)
 
+	noexpiry := GetSpecificIDFromTerraResults(2, create)
```

Comment:
If I'm understanding this correctly, we are trying to pull out the 2nd resource from the terraform result by index?
Should this be this instead?
```suggestion
	noexpiry := GetSpecificIDFromTerraResults(1, create)
```

Reply:
No, the input is the length of the resultant matched id's not the index.